Gemfile.lock

```diff
@@ -87,7 +87,7 @@ GEM
       activesupport (>= 4.2.0)
     i18n (1.8.5)
       concurrent-ruby (~> 1.0)
-    jbuilder (2.10.0)
+    jbuilder (2.10.1)
```

Comment:
### [jbuilder](https://github.com/rails/jbuilder)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.10.0 → 2.10.1

[_~~change-log~~, [source-code](https://github.com/rails/jbuilder)_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/rails/jbuilder/compare/v2.10.0...v2.10.1).

These are the first 10 commits:
- (4365ab4) [namespaced_file_path was removed from railties](https://github.com/rails/jbuilder/commit/4365ab4569d80040def8d686f61606df5540a2a4)
- (edbb333) [Merge pull request #471 from fnix/namespace-method](https://github.com/rails/jbuilder/commit/edbb33391977404a9ae0678361f77e497e8805ff)
- (b060d41) [Drop a badge which is no longer active](https://github.com/rails/jbuilder/commit/b060d41baf0c7ef004dd85b95ac5b0d3f8d29959)
- (187ee76) [Merge pull request #478 from olleolleolle/patch-2](https://github.com/rails/jbuilder/commit/187ee76493de67ac66407f43abf4ea55a1dbcaf1)
- (8bb25c2) [Gemfile for 5.2: use 5-2-stable branch](https://github.com/rails/jbuilder/commit/8bb25c2d429ea9de8d0b7340cf49dca19f117997)
- (8fd9530) [CI: Set dist: trusty for Rubinius](https://github.com/rails/jbuilder/commit/8fd95307078ab1a792f17e1f60a2c90bd6db59bf)
- (374a7f3) [Test helper: Use new Mocha setup method](https://github.com/rails/jbuilder/commit/374a7f3823df909eeaece68220624d22f9eedaff)
- (50afac4) [gemspec: Rubinius support gems](https://github.com/rails/jbuilder/commit/50afac42a01cae52da9d131673974db92909a3b1)
- (2cb1ac0) [Merge pull request #480 from olleolleolle/try-with-5-2-stabl…](https://github.com/rails/jbuilder/commit/2cb1ac069a3502715a14fce7162bc8c240782881)
- (2dd7846) [Merge pull request #482 from olleolleolle/avoid-mocha-depr-w…](https://github.com/rails/jbuilder/commit/2dd7846d803eb6cb609ff98c58906faf74633c02)


</details>